staging/src/k8s.io/cloud-provider/features/gce.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package features
+
+import (
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+)
+
+// TODO: this file should ideally live in k8s.io/cloud-provider-gcp, but it is
+// temporarily placed here to remove dependencies to k8s.io/kubernetes in the
+// in-tree GCE cloud provider. Move this to k8s.io/cloud-provider-gcp as soon
+// as it's ready to be used
+const (
+	// owner: @verult
+	// GA: v1.13
+	//
+	// Enables the regional PD feature on GCE.
+	GCERegionalPersistentDisk utilfeature.Feature = "GCERegionalPersistentDisk"
```

Comment:
I understand this is not the ideal place for this, but this would unblock the cloud provider extraction work and I think it's worth the trade-off, happy to consider alternatives though.

Reply:
+1

Reply:
This dependency on k8s.io/apiserver is really not what we want. Can we move the generic feature gate code (not the actual instance) to component-base? /cc @luxas @mtaufen @stewart-yu

Reply:
No objections from me :)

Reply:
Sorry for delay, back from Spring Festival now.
and no objections yet

Reply:
Awesome, thanks for following up :). I'll see if I can get around to this in the next few days